src/GRA.Controllers/ProfileController.cs

```diff
@@ -24,13 +24,17 @@ public class ProfileController : Base.UserController
     {
         private const string ActivityMessage = "ActivityMessage";
         private const string SecretCodeMessage = "SecretCodeMessage";
+        private const int BadgeListTake = 18;
```

Comment:
Why 18? Can this be set somewhere so it's not a magic number in a one-off controller?

Reply:
Fixed in b929d6d9e2a5141c39506733230f982c08a4cf6e

Reply:
Was 18 for compatibility with bootstrap 3's grid system with 6 badges per row. The margins are uneven now with 5 across since it doesn't fit into 12. Is there a better place to put that constant?